superset-frontend/src/dashboard/components/BuilderComponentPane.jsx

```diff
@@ -47,7 +47,7 @@ class BuilderComponentPane extends React.PureComponent {
   renderTabs(height) {
     const { isSticky } = this.props;
     return (
-      <Tabs className="m-t-10 tabs-components">
+      <Tabs className="m-t-10 tabs-components" id="tabs">
```

Comment:
Doesn't seem objectionable, but I'm curious what this fixes!

Reply:
It's required by Tabs component, otherwise this warning is generated:
![image](https://user-images.githubusercontent.com/15073128/94421688-36b4dc00-0186-11eb-9b9e-6cf841015905.png)

Reply:
Ahh, ok, thanks :)